module.go

```diff
@@ -496,9 +512,14 @@ func shouldReadBody(req *http.Request, m *Module) bool {
 		return false
 	}
 
-	// skip reading if post is invalid or too long
-	if req.ContentLength <= 0 || req.ContentLength > m.maxContentLength {
-		return false
```

Comment:
How might chunked requests be handled - there may not be a content-length header in the request?

Reply:
I believe chunked is similar - no C-L. So this *could* be used for chunked as well (need to test), but I am not sure we want to do that unless there are limits in place. Right now in gRPC you can set limits on the message size. The problem in general is:

If we read from a stream (chunked, say) and get to the max-limit, then what do we do? Right now we read it all in, then re-add the buffer as a reader injected into the request. But if we read up to max, then decide it is too big, then how to re-inject the content? Need something smarter to do that (a later enhancement) that reads from the buffer already read, then when that is finished reads more from the io.Reader (or similar) that can be re-injected. But that is going to take some more thought and testing that I was not wanting to do for this PoC.

Reply:
👍